src/app/CommandHandler.h

```diff
@@ -57,6 +57,30 @@
 namespace chip {
 namespace app {
 
+/// Defines an abstract class of something that can be encoded
+/// into a TLV with a given data tag
+class EncoderToTLV
+{
+public:
+    virtual ~EncoderToTLV() = default;
+
+    virtual CHIP_ERROR Encode(TLV::TLVWriter &, TLV::Tag tag) = 0;
+};
+/// Defines an abstract class of something that can be encoded
```

Comment:
Does this comment go with the DataModelEncoderToTlv class? I think this is meant to say concrete (or just not abstract)

Reply:
Ah, copy and paste got me (even though I was convinced I typed a proper comment). Updated.